src/server/api/models/Theme.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  Model,
+  Table,
+  Column,
+  DataType,
+  AutoIncrement,
+  PrimaryKey,
+  AllowNull,
+} from 'sequelize-typescript';
+
+type ThemeAttributes = {
+  id: number;
+  name: string;
+  style: string;
```

Comment:
Здесь бы надобно тоже Record, непонятно как там дальше это будет при непосредственной работе с моделями резолвится
```suggestion
  style: Record<string, unknown>;
```